.beachballrc

```diff
@@ -2,5 +2,5 @@
     "branch": "dev",
     "access": "public",
     "changehint": "Need to run the following command before merging this PR: npm run beachball:change",
-    "scope": ["lib/**", "extensions/**"]
+    "scope": ["lib/**", "extensions/*"]
```

Comment:
Why can lib use `**` but extensions can only do `*`?

Reply:
We can use `*` for lib too, I'll update it for consistency. I changed extensions because it was picking up the samples as well